src/components/Profile/MyPosts/Post/Post.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import s from "./Post.module.css";
+
+const AvatartsLink = "https://steamcdn-a.akamaihd.net/steamcommunity/public/images/avatars/86/86ee5650444ed1a82866aea7e6ce7e86aa3d560d_full.jpg"
```

Comment:
Consider changing the case of "AvatarLink" to upper case like so "AVATAR_LINK".
Check the spelling, mind the typing mistakes.